### DISCUSSION TITLE
注意看! 中國 App 太狠啦，毫不留情霸榜美國應用程式下載排行

### DISCUSSION THREAD

**Discussion by zephyrxin01** [1101-GS4719-C/Python-Community-GS4719-C#1385] [General]
- [【美國焦慮啊】被中國 App 霸榜！中系 App 強力輸入美國的 2 個秘密是什麼？](https://buzzorange.com/techorange/2023/03/31/us-anxiety-toward-chinese-tech/)
- 文章摘要
    - 打開美國市場的應用程式下載排行，你很快就會發現盤踞前幾名的 App，母公司幾乎全都來自中國：
        - 在蘋果 App Store 最多人下載的免費 App 分別為Temu(海外版拼多多)、TikTok、CapCut
        - 在 Google Play Store，前 3 名則是 Temu、TikTok、Shein
        - 這些受歡迎的 App 都由中國社群媒體或跨境電商公司推出。排行榜第5名之後的才是美國人自家的 Facebook。
    - 專業人士分析成功的原因歸功於?
        - 利用中國10億網路使用者測試用戶偏好、不斷優化、產生超強AI模型和演算法機制
        - 砸重金進行網路行銷
        - 狼性超強的職場文化與美國競業都自嘆弗如的高效率工作環境
            >不知道為什麼在這邊想到台積電
- 討論
    - 你們覺得中國app成功的關鍵要素是什麼呢?你贊同文章剖析出的那幾點要素嗎?
    - 不知道大家有沒有用過tiktok或是其他中國app? 我覺得滿神奇的是身邊很多人會揶揄看抖音的人，但其實youtube或是instagram也出現了很多短影片的推薦，甚至比例有越來越高的趨勢(前陣子很流行的:注意看這個男人太狠了...)。對於這樣的趨勢，你們有什麼看法嗎?之前有看到一些研究在分析這個世代接收資訊的方式已經逐漸碎片化，越來越少人能夠靜下來去看一本書甚至是一部電影。對我來說的確影響滿大，但我也仍在調適這樣的變化。像是今天如果有一個社會事件發生，我通常最先想要看到的就是相關的「懶人包」，如果感到興趣才可能去挖掘整件事情的脈絡。但其實有可能自己對整件事情的觀感可能會從最一開始的懶人包就被影響了，甚至逐漸缺乏自己去思考、剖析、媒體識讀的能力。

**Comment by ziihui** [comment]
中國App成功的關鍵要素可能是多元的，但其中一個重要的因素是在用戶偏好和市場需求方面的不斷優化。這些公司在中國網路使用者眾多的市場上有很好的測試和實驗環境，能夠快速獲得反饋並不斷調整和改進產品，從而建立了強大的AI模型和演算法機制。此外，也有可能因為狼性超強的職場文化和高效率工作環境，使得這些公司能夠更快地推出產品和服務，並迅速滿足用戶需求。

對於影響碎片化和資訊過度簡化的問題，這是一個值得關注和深思的問題。碎片化的資訊傳遞方式可能會讓人失去整體思考能力和深度閱讀能力，但也有可能提高資訊的普及率和易於理解程度，讓更多人能夠參與討論和分享。關鍵在於如何平衡碎片化和深度思考的能力，這需要個人和社會各方面的努力和配合。

**Comment by a3240281370** [comment]
但這也讓我思考到資安的問題，像是前陣子TikTok被指帶來風險及面臨美國封殺，來自中國的App攻占美國及全球市場，帶來前所未有的資安風險疑慮。而現在有許多中國的程式為了避免被其他國家封殺，表面上對「去中國化」這部分做了很多努力，實際上對資安問題仍有待商榷

**Comment by youtinglin** [comment]
我認為他們app功能強大且準確是因為他們背後有強大的資料庫，畢竟人口多，也容易蒐集到個資，因此建置的模型能更穩定且準確，之前就有去過中國，發現他們車子的導航系統超級強，能告訴你現在塞車預估可能塞多久，因為他們每台車的導航系統都是統一的，因此藉由資料廣泛的蒐集和計算，能更精確的計算和評估。
